app/models/vocabulary.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+###
+# @description: Represents a vocabularie to be used in a term for
+#   a specification.
+#   - A term can have many vocabularies associated. And a vocabulary
+#   may be in use by many terms.
+#   - A vocabulary will specify the meanings for a term definition.
+#   The implementation will be all in the content attribute as a Hash.
+###
+class Vocabulary < ApplicationRecord
+  belongs_to :organization
+  has_and_belongs_to_many :terms
+
+  validates :name, presence: true, uniqueness: true
```

Comment:
I believe names should be unique per organization, not globally.

Reply:
Done